curiouslearning/FeedTheMonsterJS
# Fix: Button click audio plays even if there is no button element present on the screen. (FM-309)

# Changes
- Make sure that the button click sound will only play if the button element is present on level selection screen.

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- make sure the button sound click is not playing when the left or right button is not present.

Ref: [FM-309](https://curiouslearning.atlassian.net/browse/FM-309)


[FM-309]: https://curiouslearning.atlassian.net/browse/FM-309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
